apps/src/templates/MultiCheckboxSelector.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import Radium from 'radium';
+import Immutable from 'immutable';
+import color from '../color';
+
+const MARGIN = 10;
+export const styles = {
+  header: {
+    color: color.purple,
+    fontWeight: 'normal',
+    borderBottomWidth: 1,
+    borderBottomStyle: 'solid',
+    borderBottomColor: color.purple,
+  },
+  checkbox: {
+    marginRight: MARGIN,
+    marginTop: 0,
+  },
+  selectAllCheckbox: {
+    position: 'relative',
+    bottom: 4,
+  },
+  list: {
+    marginLeft: 0,
+  },
+  listItem: {
+    listStyleType: 'none',
+    display: 'flex',
+    alignItems: 'center',
+    marginBottom: MARGIN,
+  },
+};
+
+const MultiCheckboxSelector = Radium(React.createClass({
+  propTypes: {
+    header: React.PropTypes.node,
+    selected: React.PropTypes.array,
+    items: React.PropTypes.array.isRequired,
+    onChange: React.PropTypes.func.isRequired,
+    children: React.PropTypes.element,
+    itemPropName: React.PropTypes.string,
+    style: React.PropTypes.any,
```

Comment:
well, since it's Radium, it can be whatever Radium accepts... which is based on what radium plugins are configured. So I figured I would just put any.